weave/trace/weave_client.py

```diff
@@ -965,11 +967,17 @@ def create_call(
         unbound_op = maybe_unbind_method(op)
         op_def_ref = self._save_op(unbound_op)
 
-        inputs_redacted = redact_sensitive_keys(inputs)
+        inputs_sensitive_keys_redacted = redact_sensitive_keys(inputs)
+
+        if should_redact_pii():
+            prepared_inputs = redact_pii(inputs_sensitive_keys_redacted)
```

Comment:
if this is heavy, it should be backgrounded by putting it into `send_start_call`